packages/bot-skeleton/src/scratch/blocks/Binary/Tick Analysis/tick.js

```diff
@@ -49,5 +49,7 @@ Blockly.Blocks.tick_string = {
     onchange: Blockly.Blocks.tick.onchange,
 };
 
-Blockly.JavaScript.tick = () => ['Bot.getLastTick(false, false)', Blockly.JavaScript.ORDER_ATOMIC];
+Blockly.JavaScript.tick = () => ['Bot.getLastTick(false, true)', Blockly.JavaScript.ORDER_ATOMIC];
```

Comment:
@sara-fs I am not sure about this change according to the getLastTick function it returns should return a string if the second parameter is being true and could you please `sum` tick with a number and check if it works fine?

Reply:
So we can add toFixed to both and convert one of them to a number